.eslintrc.json

```diff
@@ -2,16 +2,16 @@
   "extends": ["airbnb", "airbnb/hooks", "prettier"],
```

Comment:
As I said before, all this rules are not applying because it only gets one config file.